src/zope/dublincore/dcterms.py

```diff
@@ -92,96 +101,99 @@ def check_period(value):
         if "end" in d:
             check_w3cdtf(d["end"])
 
+
 def check_w3cdtf(value):
     pass
 
+
 def check_rfc3066(value):
     pass
 
+
 encodings = {
     # name --> (allowed for, validator|None),
-    "LCSH":     (("Subject",), None),
-    "MESH":     (("Subject",), None),
-    "DDC":      (("Subject",), None),
-    "LCC":      (("Subject",), None),
-    "UDC":      (("Subject",), None),
+    "LCSH": (("Subject",), None),
+    "MESH": (("Subject",), None),
+    "DDC": (("Subject",), None),
+    "LCC": (("Subject",), None),
+    "UDC": (("Subject",), None),
     "DCMIType": (("Type",), check_dcmitype),
-    "IMT":      (("Format",), check_imt),
+    "IMT": (("Format",), check_imt),
     "ISO639-2": (("Language",), check_iso639_2),
-    "RFC1766":  (("Language",), check_rfc1766),
-    "URI":      (("Identifier", "Relation", "Source",), check_uri),
-    "Point":    (("Coverage.Spatial",), check_point),
-    "ISO3166":  (("Coverage.Spatial",), check_iso3166),
-    "Box":      (("Coverage.Spatial",), check_box),
-    "TGN":      (("Coverage.Spatial",), check_tgn),
-    "Period":   (("Coverage.Temporal",), check_period),
-    W3CDTF:     (("Coverage.Temporal", "Date",), check_w3cdtf),
-    "RFC3066":  (("Language",), check_rfc3066),
-    }
+    "RFC1766": (("Language",), check_rfc1766),
+    "URI": (("Identifier", "Relation", "Source",), check_uri),
+    "Point": (("Coverage.Spatial",), check_point),
+    "ISO3166": (("Coverage.Spatial",), check_iso3166),
+    "Box": (("Coverage.Spatial",), check_box),
+    "TGN": (("Coverage.Spatial",), check_tgn),
+    "Period": (("Coverage.Temporal",), check_period),
+    W3CDTF: (("Coverage.Temporal", "Date",), check_w3cdtf),
+    "RFC3066": (("Language",), check_rfc3066),
+}
```

Comment:
PEP-8 is **wrong** to forbid column alignment, and that's an opinion I'll carry to my grave.